build-openjdk11.sh

```diff
@@ -34,6 +34,8 @@ TARBALL_VERSION="${UPDATE}_${BUILD}${EA_SUFFIX}"
 PLATFORM_VERSION="${PLATFORM}_${TARBALL_VERSION}"
 TARBALL_NAME="${TARBALL_BASE_NAME}-jdk_${PLATFORM_VERSION}"
 TARBALL_NAME_JRE="${TARBALL_BASE_NAME}-jre_${PLATFORM_VERSION}"
+TARBALL_NAME_SHENDOH="${TARBALL_BASE_NAME}-jdk-shenandoah_${PLATFORM_VERSION}"
+TARBALL_NAME_SHENDOH_JRE="${TARBALL_BASE_NAME}-jre-shenandoah_${PLATFORM_VERSION}"
```

Comment:
`TARBALL_NAME_SHENDOH` seems odd. Could we use `TARBALL_NAME_SHENANDOAH` instead for clarity, please?